pkg/detectors/fibery/fibery.go

```diff
@@ -0,0 +1,88 @@
+package fibery
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"regexp"
+	"strings"
+	"time"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+type Scanner struct{}
+
+// Ensure the Scanner satisfies the interface at compile time
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	client = common.SaneHttpClient()
+
+	//Make sure that your group is surrounded in boundry characters such as below to reduce false positives
+	keyPat    = regexp.MustCompile(detectors.PrefixRegex([]string{"fibery"}) + `\b([0-9a-z.]{44})\b`)
```

Comment:
suggestion: I think we can make this a little bit more strict `\b([0-9a-f]{8}.[0-9a-f]{35})\b` all the keys are encoded so we can only work with `a-f` additionally they are all split into two section by a `.` separator. So we can use that to our advantage.